### PR TITLE
add jwt to the project

### PR DESCRIPTION
**Pull request by Bohadan-Markatov**

### CODE REVIEW

File: src/main/java/mate/academy/bookshop/util/JwtUtil.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    public boolean isValidToken(String token) {
+        try {
+            Jws<Claims> claimsJws = Jwts.parser()
+                    .setSigningKey(key)
+                    .build()
+                    .parseClaimsJws(token);
+
+            return !claimsJws.getBody().getExpiration().before(new Date());
+        } catch (Exception e) {
+            throw new JwtException("Expired or invalid JWT token", e);
```
Comment (vovasalyha) [inline src/main/java/mate/academy/bookshop/util/JwtUtil.java]:
```suggestion
            return false;
```

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
+spring.datasource.url=jdbc:mysql://localhost:3306/book?Timezone=Europe/Kiev
+spring.datasource.username=root
+spring.datasource.password=1234
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.show-sql=true
+jwt.expiration=100000
+jwt.secret=djflsagdkfjgkfdjnjgkdjdjsdjsjjdsijfdsls
```
Comment (vovasalyha) [inline src/main/resources/application.properties]: It's better to get sensitive properties from system properties and not pushing it to git.

**Review by vovasalyha — CHANGES_REQUESTED** [review]
Looks good, but see a couple of comments.

**Review by d1sam — APPROVED** [review]
Well done!

**Pull request merged**